src/main/java/uk/gov/hmcts/reform/hmc/controllers/HearingManagementController.java

```diff
@@ -130,10 +132,31 @@ public GetHearingsResponse getHearings(@PathVariable("ccdCaseRef")
     public HearingResponse updateHearing(@RequestBody @Valid UpdateHearingRequest hearingRequest,
                                          @PathVariable("id") Long hearingId) {
         HearingResponse hearingResponse = hearingManagementService.updateHearingRequest(hearingId, hearingRequest);
-        hearingManagementService.sendRequestToHmiAndQueue(hearingId, hearingRequest, AMEND_HEARING);
+        hearingManagementService.sendRequestToHmiAndQueue(hearingRequest, hearingId, AMEND_HEARING);
         return hearingResponse;
     }
 
+
+
+    @PostMapping(path = "/test", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.CREATED)
+    public HmiSubmitHearingRequest test(@RequestBody CreateHearingRequest createHearingRequest) {
+        hearingManagementService.verifyAccess(createHearingRequest.getCaseDetails().getCaseRef());
+        HearingResponse hearingResponse = hearingManagementService.saveHearingRequest(createHearingRequest);
+        return hearingManagementService.test(
+                hearingResponse.getHearingRequestId(),
+                createHearingRequest
+        );
+    }
+
+    @PutMapping(path = "/test/{id}", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.CREATED)
+    public HmiSubmitHearingRequest test(@RequestBody UpdateHearingRequest hearingRequest,
+                                        @PathVariable("id") Long hearingId) {
+        hearingManagementService.updateHearingRequest(hearingId, hearingRequest);
+        return hearingManagementService.test(hearingId, hearingRequest);
+    }
```

Comment:
Can we delete these temporary test methods? I don't think we need them anymore since we now have the functionality in place to put messages on the queue. We can test them properly now.

Reply:
Yup. Done. Padma showed me how to use the queues instead.